Makefile

```diff
@@ -0,0 +1,29 @@
+test:
+	go test -v ./...
+
+build: test
+	go build
+
+check: lint vet fmtcheck ineffassign
+
+lint:
+	golint -set_exit_status .
+
+vet:
+	go vet
+
+fmtcheck:
+	@ export output="$$(gofmt -s -d .)"; \
+		[ -n "$${output}" ] && echo "$${output}" && export status=1; \
+		exit $${status:-0}
+
+ineffassign:
+	ineffassign .
+
+setup:
+	go get github.com/Masterminds/glide
```

Comment:
I'd rather avoid these external dependencies for now.

Reply:
That was a mistake for my part, sorry: glide is not needed (This is a leftover from copy/pasting).

If I get rid of ineffassign, this will just leave go-getting golint. Still unacceptable ? FYI this will only be called in travis and users of `ng` won't be impacted by this.